Modules/Sources/Yosemite/PointOfSale/Coupons/PointOfSaleCouponFetchStrategyFactory.swift

```diff
@@ -6,7 +6,12 @@ import class Networking.AlamofireNetwork
 import struct Combine.AnyPublisher
 import struct NetworkingCore.JetpackSite
 
-public struct PointOfSaleCouponFetchStrategyFactory {
+public protocol PointOfSaleCouponFetchStrategyFactoryProtocol {
```

Comment:
Note: a protocol was created so that `StorageManagerType`, a dependency of `PointOfSaleCouponFetchStrategyFactory`, does not need to be known in the future POS module.